changelogs/fragments/5206-proxmox-conditional-vmid.yml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - proxmox_kvm - replace new condition with proper condition to allow for using ``vmid`` on update (https://github.com//pull/5206).
```

Oops, part of the URL was lost along the path.
```suggestion
  - proxmox_kvm - replace new condition with proper condition to allow for using ``vmid`` on update (https://github.com/ansible-collections/community.general/pull/5206).
```

Also, "replace new condition with proper condition" sounds a little bit odd. Any better way to describe it, by any chance?

---

There recently was a bug in GitHub that screwed up URLs when merging suggestions for that line. Thanks for catching this!